devel/asu/files/asu.init

```diff
@@ -23,6 +23,8 @@ start_service() {
       [ $((i++)) -gt 30 ] && return 1
     done
 
+    cp /etc/asu/odbc.ini /var/run/asu/.odbc.ini
```

Comment:
@dangowrt is the dest folder already created at this point?

Reply:
`/var` is tmpfs, so you do need to create the folder before using `mkdir -p`.